src/component/Install.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {Button, Datepicker, Select} from '@datorama/app-components';
+import sdk from '@datorama/sdk';
+import styled from "styled-components";
+import {useDispatch, useSelector} from 'react-redux';
+import {changeDataStream, changeDateRange} from '../redux/actions/todo.actions';
+import moment from "moment";
+
+const Install = () => {
+    const dispatch = useDispatch();
+    const installConfig = useSelector(state => state.todo);
+
+    const handleClick = () => {
+        console.log(installConfig);
+        sdk.app.install({config: installConfig});
+    };
+
+    const [selected, setSelected] = useState([]);
+    const handleDateRangeClick = (event) => {
+        dispatch(changeDateRange(event));
+    };
+
+    const handleDataStreamClick = event => {
+        setSelected(event);
+        dispatch(changeDataStream(event[0]));
+    };
+
+
+    const options = [
+        {value: '1', label: 'Facebook'},
+        {value: '2', label: 'Google Analytics'},
+        {value: '3', label: 'MailChimp'},
+        {value: '4', label: 'Instagram'},
+        {value: '5', label: 'Google Ads'},
+    ];
+
+    const dateRangePicker = {
+        startDate: moment().toDate(),
+        endDate: moment()
+            .add(3, 'days')
+            .toDate()
+    };
+
+    return (
+        <Container>
+            <Header top={-130} left={-160}>
+                Installation Page
+            </Header>
+            <Position top={-70} left={-189}>
```

Comment:
same here for the styles. i know that it's just a demo project, but in the next sprint ill do a small CSS workshop and you'll see that it's easy to work with flexbox for example.

try this one if you have time - https://flexboxfroggy.com/